suite-native/analytics/src/events.ts

```diff
@@ -182,6 +182,7 @@ export type SuiteNativeAnalyticsEvent =
           payload: {
               discoveryId: string; // Used for grouping multiple events of a single discovery run together.
               loadDuration: number;
+              networkSymbols: NetworkSymbol[];
```

Comment:
Please reflect the change here https://www.notion.so/satoshilabs/da354c80a23a44a2814748968f701c5a?v=6a0a9e88bd04494db46c139336122d37&pvs=4 and here https://www.notion.so/satoshilabs/Changelog-Mobile-app-946307159a294bcdb90b94e90eefd292?pvs=4 and ping @kolomaznikt

Reply:
done